### ISSUE TITLE
Accessibility: Certain information about a message thread is not accessible straight away with a screen reader

### ISSUE THREAD

**Issue by Masarczyk-Wienfluss** [mattermost/mattermost#26594]
Per Mattermost guidelines, GitHub issues are for bug reports: <https://handbook.mattermost.com/contributors/contributors/ways-to-contribute/>.

For troubleshooting see: https://forum.mattermost.com/.
For feature proposals see: https://mattermost.com/suggestions/

If you've found a bug--something appears unintentional--please follow these steps:

1. Confirm you’re filing a new issue. [Search existing tickets in Jira](https://mattermost.atlassian.net/jira/software/c/projects/MM/issues/) to ensure that the ticket does not already exist.
2. Confirm your issue does not involve security. Otherwise, please see our [Responsible Disclosure Policy](https://mattermost.com/security-vulnerability-report/).
3. [File a new issue](https://github.com/mattermost/mattermost/issues/new) using the format below. Mattermost will confirm steps to reproduce and file in Jira, or ask for more details if there is trouble reproducing it. If there's already an existing bug in Jira, it will be linked back to the GitHub issue so you can track when it gets fixed. 

#### Summary
Bug report in one concise sentence
Information such as the number of unread messages in a thread, number of replies or reactions of a certain type is not reported while focusing a message in a conversation.
#### Steps to reproduce
How can we reproduce the issue (what version are you using?)
1. Load any conversation.
2. With a screen reader such as NVDA on, navigate to the message list using F6.
3. Focus a message that has been replied to thus creating a thread or that there have been reactions to.
#### Expected behavior
Describe your issue in detail

The key information such as number of replies, reactions of a certain type and number of unread messages, if the message is a thread starter, is reported back.
#### Observed behavior (that appears unintentional) 
What did you see happen? Please include relevant error messages and/or screenshots.
The key information such as the number of replies, reactions of a certain type and the number of unread messages if the message is a thread starter are not reported back.

#### Possible fixes
If you can, link to the line of code that might be responsible for the problem
The information in question seems to be reachable by repeatedly pressing the Tab key. Possibly, when navigating with arrow keys, those links could be also included as a part of the focused and read out region.

**Comment by amyblais** [comment]
Ticket: https://mattermost.atlassian.net/browse/MM-58011
